src/client.ts

```diff
@@ -495,13 +497,15 @@ export function createClient(options: ClientOptions): Client {
             // either the canceller will be called and the promise resolved
             // or the socket closed and the promise rejected
             await throwOnCloseOrWaitForCancel(() => {
-              // send complete message to server on cancel
-              socket.send(
-                stringifyMessage<MessageType.Complete>({
-                  id: id,
-                  type: MessageType.Complete,
-                }),
-              );
+              if (!completed && socket.readyState === WebSocketImpl.OPEN) {
```

Comment:
I added this check for the socket state because I noticed some errors in my console where socket state was already CLOSING or CLOSED at this point.

Reply:
Crazy, I just left a comment about that here: https://github.com/enisdenjo/graphql-ws/pull/65#discussion_r522142673 😅 . This has to be handled then, the socket _should_ be still open for the cleanup.